### PR TITLE
Diff production model

### PR DESCRIPTION
**Pull request by katrilh**
## Description
Please describe the change you have made.

## Checklist:
- [x] Tests added/updated.
- [x] Documentation updated.
- [x] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/power-ops-sdk/blob/master/CHANGELOG.md).
- [x] Version bumped. If triggering a new release is desired, bump the version number in [pyproject.toml](https://github.com/cognitedata/power-ops-sdk/blob/master/pyproject.toml) per [semantic versioning](https://semver.org/).

### CODE REVIEW

File: cognite/powerops/resync/models/market/dayahead.py [hunk 1]
```diff
@@ -30,6 +30,17 @@ class DayAheadProcess(Process):
...
+    # # i teorien, løftes til AssetType... kanskje
+    # def from_cdf(
+    #     cls,
+    #     client,
+    #     external_id: str,
+    #     fetch_relationships: bool = False,
+    #     fetch_content: bool = False,
+    # ) -> DayAheadProcess:
+    #     raise NotImplementedError()
+
```
Comment (katrilh) [inline cognite/powerops/resync/models/market/dayahead.py:43]: Tanken med å la denne bli værene er å sendere kunne teste modellene på market, ikke kun produksjon

**Comment by Juliamg** [comment]
Great work! Readable and easy to follow 😄 

Still getting the weird diff print with this code, though so we will have to look into that. Ex.:

Difference for resource type assets: 
--------------------------------------------
Value of root['POWEROPS_Fornebu_week_up_day_3-prices_20MW-200MW']['metadata']['bid:price_scenarios'] changed from "['cognite', '-20pct', '+20pct']" to "["cognite", "-20pct", "+20pct"]".
Value of root['POWEROPS_Fornebu_week_up_day_3-prices_20MW-200MW']['metadata']['bid:reserve_scenarios'] changed from "[0mw, 20mw, 40mw, 60mw, 80mw, 100mw, 120mw, 140mw, 160mw, 180mw, 200mw]" to "["0mw", "20mw", "40mw", "60mw", "80mw", "100mw", "120mw", "140mw", "160mw", "180mw", "200mw"]".
Value of root['POWEROPS_Fornebu_week_up_day_3-prices_20MW-200MW']['metadata']['rkom:plants'] changed from "['dalby', 'holen', 'landet', 'rull1', 'rull2', 'scott']" to "["dalby", "holen", "landet", "rull1", "rull2", "scott"]".
Value of root['POWEROPS_Fornebu_week_down_day_3-prices_20MW-200MW']['metadata']['bid:price_scenarios'] changed from "['cognite', '-20pct', '+20pct']" to "["cognite", "-20pct", "+20pct"]".
Value of root['POWEROPS_Fornebu_week_down_day_3-prices_20MW-200MW']['metadata']['bid:reserve_scenarios'] changed from "[0mw, 20mw, 40mw, 60mw, 80mw, 100mw, 120mw, 140mw, 160mw, 180mw, 200mw]" to "["0mw", "20mw", "40mw", "60mw", "80mw", "100mw", "120mw", "140mw", "160mw", "180mw", "200mw"]".
Value of root['POWEROPS_Fornebu_week_down_day_3-prices_20MW-200MW']['metadata']['rkom:plants'] changed from "['dalby', 'holen', 'landet', 'rull1', 'rull2', 'scott']" to "["dalby", "holen", "landet", "rull1", "rull2", "scott"]".
Value of root['POWEROPS_Fornebu_weekend_up_day_3-prices_20MW-200MW']['metadata']['bid:price_scenarios'] changed from "['cognite', '-20pct', '+20pct']" to "["cognite", "-20pct", "+20pct"]".
Value of root['POWEROPS_Fornebu_weekend_up_day_3-prices_20MW-200MW']['metadata']['bid:reserve_scenarios'] changed from "[0mw, 20mw, 40mw, 60mw, 80mw, 100mw, 120mw, 140mw, 160mw, 180mw, 200mw]" to "["0mw", "20mw", "40mw", "60mw", "80mw", "100mw", "120mw", "140mw", "160mw", "180mw", "200mw"]".
Value of root['POWEROPS_Fornebu_weekend_up_day_3-prices_20MW-200MW']['metadata']['rkom:plants'] changed from "['dalby', 'holen', 'landet', 'rull1', 'rull2', 'scott']" to "["dalby", "holen", "landet", "rull1", "rull2", "scott"]".

**Review by Juliamg — APPROVED** [review]

File: cognite/powerops/resync/models/helpers.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+
+from cognite.client.data_classes import Relationship
+
+
+def isinstance_list(value: Any, type_: Type):
+    return isinstance(value, list) and value and isinstance(value[0], type_)
+
+
+def match_field_from_relationship(model_fields: list[str], relationship: Relationship) -> str:
```
Comment (haukb) [inline cognite/powerops/resync/models/helpers.py:18]: could benefit from a docstring, to understand what is does without having to read :)

**Review by haukb — APPROVED** [review]
Perfect! Superb work Katie, one minor comment, you can do it if you think it makes sense **and** you have time

**Pull request merged**